### PR TITLE
vet clinic database

### PR DESCRIPTION
**Pull request by ijosue16**
In this PR I did the following:

- [ ] created a vet clinic database using Postgres.
- [ ] created a table called animals with its column elements.
- [ ] added data to the animal table.
- [ ] wrote queries to retrieve data due to given conditions.

##Here are the query results screenshots
create table

Screenshot (18)
Screenshot (11)
Screenshot (12)
Screenshot (20)
Screenshot (14)
Screenshot (15)
Screenshot (16)
Screenshot (17)

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Iradukunda, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe. Similarly, kindly also remove guide notes for _license_ creation under the _license_ section.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
+-- create table
 CREATE TABLE animals (
-    name varchar(100)
-);
+id BIGSERIAL NOT NULL,
```
Comment (Deepakdanger) [inline schema.sql:5]: - [x] Kindly also mark `id` as primary key

File: schema.sql [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+name VARCHAR(100),
+date_of_birth DATE,
+escape_attempts INT,
+neutered BOOLEAN,
+weight_kg FLOAT(30));
```
Comment (Deepakdanger) [inline schema.sql]: - [x] Here `weight_kg` should be type `DECIMAL`.Kindly change it

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @ijosue16 , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 2 above)
Comment (ijosue16) [reply schema.sql]: all changes applied using queries


Screenshot (23)

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @ijosue16 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlights 
:+1: no linter errors 
:+1: Changes requested by the last reviewer implemented 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**